src/site/content/en/blog/webcodecs/index.md

```diff
@@ -145,46 +159,51 @@ let config = {
   codec: 'vp8',
   width: 640,
   height: 480,
-  bitrate: 8_000_000,     // 8 Mbps
+  bitrate: 2_000_000, // 2 Mbps
   framerate: 30,
 };
 
 let encoder = new VideoEncoder(init);
 encoder.configure(config);
 ```
 
-After the encoder has been set up,  it's ready to start accepting frames. When
-frames are coming from a media stream, the callback given to `VideoTrackReader.start()` will pump frames into
-the encoder, periodically inserting
-[keyframes](https://en.wikipedia.org/wiki/Key_frame#Video_compression) and
-checking that the encoder is not overwhelmed with incoming frames.
+After the encoder has been set up, it's ready to start accepting frames via `encode()` method.
```

Comment:
```suggestion
After the encoder has been set up, it's ready to accept frames via `encode()` method.
```

Reply:
There's a writing best practice to omit needless words, that is, words that don't contribute meaning. A common source is what's called an inflated verb. If there are two verb forms ("start accepting") it can almost always be compressed to the second (accept).